src/arlo_plugin/camera.py

```diff
@@ -578,6 +608,23 @@ async def ignore_trickle(c):
 
         return ArloCameraRTCSessionControl(plugin_session)
 
+    def parse_sdp(self, sdp):
+        lines = sdp.split('\n')
+        lines = [line.strip() for line in lines]
+
+        section = []
+
+        for line in lines:
+            if line.startswith('a=candidate:'):
+                if line.count(':') <= 1 and not ".local" in line:
```

Comment:
Please add a comment before this line documenting that we want to filter out ipv6 and local addresses

Reply:
This should be done in the latest commit now.